### PR TITLE
DDF-04321 Temporary fix for TestCatalogValidation and timeout changed to 2 mins

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Temporarily fixes the failing tests in TestCatalogValidation by resetting the state after each test. The long term fix should be having each test clean up after themselves. This also changes the timeout from 20 minutes to 2 minutes as it is only used when ingesting and changing properties. 
#### Who is reviewing it? 

@austinsteffes @brjeter @ethantmanns 
#### Select relevant component teams: 

@codice/test 
#### Ask 2 committers to review/merge the PR and tag them here.

@brjeter 
@ethantmanns

#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #04321

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [x] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java [hunk 1]
```diff
@@ -141,9 +141,9 @@
 
   private static final File UNPACK_DIRECTORY = new File("target/exam");
 
-  public static final long GENERIC_TIMEOUT_SECONDS = TimeUnit.MINUTES.toSeconds(20);
+  public static final long GENERIC_TIMEOUT_SECONDS = TimeUnit.MINUTES.toSeconds(2);
 
-  public static final long GENERIC_TIMEOUT_MILLISECONDS = TimeUnit.MINUTES.toMillis(20);
+  public static final long GENERIC_TIMEOUT_MILLISECONDS = TimeUnit.MINUTES.toMillis(2);
```
Comment (brjeter) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: It looks like these constants are used in a lot of places in the itests, so I'm a little concerned that some of the operations may not complete in 2 minutes. But I guess we can see how the CI build goes.

**Comment by brjeter** [comment]
build now

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java (hunk 1 above)
Comment (AzGoalie) [reply distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java]: We could up it to 5 minutes as `REMOVE_ALL_TIMEOUT` has it further up.

**Comment by austinsteffes** [comment]
quick build success and ran TestCatalogValidation and passed

**Pull request merged**